integreat_cms/cms/migrations/0071_update_external_news.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.18 on 2023-05-23 12:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Rename database field and verbose name of external news
+    """
+
+    dependencies = [
+        ("cms", "0070_alter_poicategory_icon"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="region",
+            name="tunews_enabled",
+        ),
+        migrations.AddField(
+            model_name="region",
+            name="external_news_enabled",
+            field=models.BooleanField(
+                default=False,
+                help_text="Enable to display external articles in addition to local news managed by the CMS",
+                verbose_name="Enable external news",
+            ),
+        ),
```

Comment:
I'm pretty sure I've explained this a few times before: Removing a field and then adding a new one will cause the saved data to be lost.
[`RenameField`](https://docs.djangoproject.com/en/3.2/ref/migration-operations/#renamefield) is what you should use instead in these cases to preserve the data.
Autogenerating migration files saves a lot of time in most cases, but it doesn't mean we don't have to check whether it does what we want it to do.